features/machine/autoscaler.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Cluster Autoscaler Tests
+
+  # @author jhou@redhat.com
+  # @case_id OCP-28108
+  @admin
+  @destructive
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I store the number of machines in the :num_to_restore clipboard
+    And admin ensures the node number is restored to "<%= cb.num_to_restore %>" after scenario
+
+    Given I clone a machineset named "machineset-clone"
+    And admin ensures "machineset-clone" machineset is deleted after scenario
+
+    # Create clusterautoscaler
+    Given I use the "openshift-machine-api" project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/cluster-autoscaler.yml |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    When I run the :delete admin command with:
+      | object_type       | clusterautoscaler |
```

Comment:
@jhou1 please use the step `I ensure...` instead.

Reply:
Oh, I can remember why I didn't use `I ensure ...` in the first place. the `clusterautoscaler` and `machineautoscaler` are custom resources, we don't have a library defined around it. I'll try implement that.

Reply:
@jhou1 [machine_autoscaler.rb](https://github.com/openshift/verification-tests/blob/master/lib/openshift/machine_autoscaler.rb) is there.  You can use that as a template for `clusterautoscaler`

Reply:
It's updated now.